cli/cmd/integration.go

```diff
@@ -279,6 +281,10 @@ func promptCreateIntegration() error {
 		return createAwsConfigIntegration()
 	case "AWS CloudTrail":
 		return createAwsCloudTrailIntegration()
+	case "AWS GovCloud Config":
+		return createAwsGovCloudConfigIntegration()
+	case "AWS GovCloud CloudTrail":
+		return createAwsGovCloudCTIntegration()
```

Comment:
nit: If we change the Integration type, would be nice to change these functions too
```suggestion
		return createAwsGovCloudCloudTrailIntegration()
```